packages/types/src/chain-adapters/cosmos.ts

```diff
@@ -1,8 +1,10 @@
 export type BuildTxInput = {
   gas: string
+  memo: string
```

Comment:
Only thought would be if we want memo required in our chain adapters sign tx interface. We could always default to empty string since it looks required by hdwallet for whatever reason.

Reply:
yeah this should be optional